iked/vroute.c

```diff
@@ -408,6 +425,10 @@ vroute_getroute(struct iked *env, struct imsg *imsg)
 		return (vroute_dodefaultroute(env, flags, addrs, type, gateway));
 	}
 #endif
+	if (type == RTM_ADD)
```

Comment:
That's new. Was there no vroute_insertroute before? Was it missing? Or was another call moved?

Reply:
was missing